es-app/src/views/gamelist/GridGameListView.cpp

```diff
@@ -20,27 +90,59 @@ FileData* GridGameListView::getCursor()
 
 void GridGameListView::setCursor(FileData* file)
 {
-	if(!mGrid.setCursor(file))
+	if(!mGrid.setCursor(file) && mGrid.getEntryCount() > 0)
 	{
-		populateList(file->getParent()->getChildrenListToDisplay());
+		populateList(file->getParent()->getChildren());
```

Comment:
I think you will need to call `getChildrenListToDisplay()`, to support filtering of entries. Why would you need to call `getChildren()` directly?

Reply:
Thanks for noticing this ;)